### ISSUE TITLE
design new home screen view

### ISSUE THREAD

**Issue by guylepage3** [blockstack/blockstack-browser#140]
Design a new home screen view with a heavy focus on apps.

**Requirements**
- [ ] my apps - a list of available apps for the user
  - [ ] hello, blockstack
  - [ ] identity
  - [ ] blockstack.org
  - [ ] settings
- [ ] blockstack logo for brand recognition since the home screen is inside of another companies product, Google Chrome. (possibly removed at a later date if we develop our own browser).

**Comment by guylepage3** [comment]
Here are all the designs from yesterday recorded in the Blockstack Slack #design channel.

![dashboard-1](https://cloud.githubusercontent.com/assets/1711854/21432959/ed125e06-c83b-11e6-92c8-494d783ccd58.png)
![dashboard-2](https://cloud.githubusercontent.com/assets/1711854/21432962/f125e472-c83b-11e6-8cbf-f741a0124c6d.png)
![dashboard-3](https://cloud.githubusercontent.com/assets/1711854/21432964/f8af8d92-c83b-11e6-9f9d-a6bf999be547.png)
![dashboard-4](https://cloud.githubusercontent.com/assets/1711854/21432966/fbd629ea-c83b-11e6-8d3d-78ab18e76aaa.png)
![dashboard-5](https://cloud.githubusercontent.com/assets/1711854/21432969/ff1b14f8-c83b-11e6-85b6-05bb4c8bdc3f.png)
![dashboard-6](https://cloud.githubusercontent.com/assets/1711854/21432973/0282d9a0-c83c-11e6-89d2-6390572f94cc.png)
![dashboard-2 5](https://cloud.githubusercontent.com/assets/1711854/21432982/08f6af14-c83c-11e6-8760-eddf55b64950.png)
![dashboard-2 6](https://cloud.githubusercontent.com/assets/1711854/21432986/0d7c278a-c83c-11e6-8872-361d28ab560f.png)
![dashboard-2 7](https://cloud.githubusercontent.com/assets/1711854/21432994/11f97024-c83c-11e6-836f-b2831a082ccd.png)
![dashboard-2 8](https://cloud.githubusercontent.com/assets/1711854/21432997/16ebdf7c-c83c-11e6-9d47-0386200fb843.png)

**Comment by guylepage3** [comment]
What was decided upon yesterday was that simplicity for the user is key. This is a users "home screen." Give them what they came for. Their apps. Make it dead simple yet unique.

**Design decisions**
Background
We applied the current Blockstack brand gradient to the final version of the home screen design. We did try a variety of backgrounds but ended up removing all elements that were detracting from the main focus of getting users to their apps.

**Written communications**
A variety of written content was applied to the designs but in the end we went with a simplicity as seen on the Apple iOS home screen. This simplicity assists users to be focusing on one thing. Their apps.

**Typography**
We applied the latest Blockstack Lato and Source Code Pro typefaces. Kept the written communications small where least important, large in most important areas, and used bolding for app names.

**Footer**
We tried a variety of logo placements but kept feeling that there was imbalance in the design. In the end the simplicity of keeping the logo small, non-invasive and away from the focal point was best. We also determined that someone should know what this is by actually saying it in the footer. Removing any confusion for users. seeing the view in screen shots and/or in their browser.

**Apps**
We added a blue halo around each app as an element of uniqueness. We kept them large to fill a majority of the screen so that users could not miss them. We wanted them to scream, "here I am, click me."

**Logo (update)**
We also discovered another design differentiator in the logo/bug. We applied a new Stereocopy styling behind the icon only to give a unique and intentional blurred effect. Also in doing this, the Blockstack brand is now 3D ready and with the correct lenses can be seen as 3D.
